docs/multicluster/user-guide.md

```diff
@@ -142,10 +142,15 @@ We recommend creating one ServiceAccount for each member for fine-grained access
        namespace: antrea-mcs-ns
```

Comment:
Could you fix the invalid link to "installation-in-a-shared-cluster" in "Deploy Mulit-cluster Controller"? BTW, the deployment guide looks complex. I think we should do a quick-start section first with the simplest deployment steps, before talking abut different options. But probably let us fix it later.